interp/src/debugger/debugger_core.rs

```diff
@@ -175,6 +198,28 @@ impl<C: AsRef<Context> + Clone> Debugger<C> {
             .collect_vec();
         self.manipulate_breakpoint(Command::Delete(parsed_bp_ids));
     }
+
+    pub fn cont(&mut self) -> StoppedReason {
+        self.is_running = true;
+        let _ = self.do_continue(); //need to error handle
+        let bps = self
+            .debugging_context
+            .hit_breakpoints()
+            .map(|x| self.grp_idx_to_name(x))
+            .collect_vec();
+        if self.interpreter.is_done() {
+            StoppedReason::Done
+        } else if !bps.is_empty() {
+            StoppedReason::Breakpoint(bps)
+        } else {
+            StoppedReason::PauseReq
+        }
+    }
+
+    pub fn pause(&mut self) {
+        self.is_running = false;
+    }
```

Comment:
Can you explain a bit more what this is actually supposed to do? Since
`self.is_running` is never read this doesn't actually do anything as far as I
can tell

Reply:
Just deleted it since I actually never use the method anywhere and just handle everything in the adapter itself!